lineman/spec-e2e/proposal_spec.coffee

```diff
@@ -21,7 +21,7 @@ describe 'Proposals', ->
     beforeEach ->
       threadHelper.loadWithActiveProposal()
 
-    iit 'successfully votes on a proposal', ->
+    it 'successfully votes on a proposal', ->
```

Comment:
Eek, whoops, heh. That would explain why our e2ez weren't running for a bit.. here's hoping this passes on master!